extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -64,12 +66,14 @@ export class ProjectsController {
 	private buildHelper: BuildHelper;
 	private buildInfo: DashboardData[] = [];
 	private publishInfo: PublishData[] = [];
+	private deployService: DeployService;
 
 	projFileWatchers = new Map<string, vscode.FileSystemWatcher>();
 
-	constructor() {
-		this.netCoreTool = new NetCoreTool();
+	constructor(private _outputChannel: vscode.OutputChannel) {
```

Comment:
I don't see `_outputChannel` being used outside of the constructor; does it need to be saved as a private var here?